docs/user_manual/calculations.md

```diff
@@ -466,8 +477,37 @@ $$
     \end{eqnarray}
 $$
 
+- Branch current with global angle: Linear WLS requires a complex current phasor. The global angle current measurement captures
```

Comment:
We are repeating : `Linear WLS requires a complex current phasor.` a lot.
We can add a text before these bullet points:

We need to convert all measurements into either complex voltage or complex current for Linear WLS.

Reply:
attempted to solve it. can you please check whether it's good enough?